tests/tasks/conftest.py

```diff
@@ -0,0 +1,77 @@
+import pytest
```

Comment:
Reviewer note: Tried to reduce size. Moved fixtures into `test/tasks/conftest.py`. Moved perturbation tests into `test/tasks/test_perturbation_expression_prediction.py`.